pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -150,27 +150,28 @@ def work(self):
                         'softban',
                         formatted='Probably got softban.'
                     )
-                with self.bot.database as conn:
-                    c = conn.cursor()
-                    c.execute("SELECT COUNT(name) FROM sqlite_master WHERE type='table' AND name='softban_log'")
-                result = c.fetchone()        
-
-                while True:
```

Comment:
Why there was a while True: ? It seems just one time run, why placed the `while` here